frontend/src/leader-list.tsx

```diff
@@ -49,16 +52,27 @@ function colorForIndex(index: number): string {
 }
 
 export function getLeaders(pulls: Pull[], extractSigsFromPull: (pull: Pull) => Signature[]) {
-   const signatures: Signature[] = pulls.flatMap(extractSigsFromPull);
-   const users = signatures.map((signature) => signature.data.user);
-   const groupedByUsers = users
-      .reduce((grouped: Map<number, SigCount>, user: SignatureUser) => {
-         const group = grouped.get(user.id) || {user, count:0};
-         group.count++;
-         grouped.set(user.id, group)
-         return grouped;
-      }, new Map())
-      .values();
-   return Array.from(groupedByUsers)
+   const recent = toDateString(new Date(Date.now() - 86400 * 1000 * DAYS_TO_EXAMINE));
```

Comment:
I'm confused by the different date formats we are using. In our DB we store dates in seconds, but for Pulldasher we are using milliseconds?